molecule/rbac-scram-custom-rhel-fips/molecule.yml

```diff
@@ -16,8 +16,8 @@ platforms:
     hostname: ldap1${BUILD_NUMBER}.confluent${BUILD_NUMBER}
     groups:
       - ldap_server
-    image: geerlingguy/docker-centos7-ansible
-    # dockerfile: ../Dockerfile-rhel-java17.j2
+    image: centos:centos8
+    dockerfile: ../Dockerfile-centos8-ldap.j2
```

Comment:
Do we need this dockerfile here? I believe this scenario does not use dockerfile on purpose to test some case

Reply:
In all rbac scenarios we need to install openldap-servers package which comes from the repository powertools. In geerlingguy/docker-centos7-ansible docker image it is readily available (i.e powertools repo is pre enabled) and hence we dont need any extra configuration in Dockerfile vs in centos8 docker image 2 additional things are required. 
- RUN yum -y config-manager --set-enabled powertools (to enable powertools repo which is by default in disabled state)
- Some commands to enable systemd (as this vanila centos8 image doesnt have systemd enabled by default)